modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -47,6 +63,58 @@ If you provision the adapter with organization credentials, you will be able to
 If you specify environment credentials, you will only be able to manage APIs of that environment. MuleSoft recommends that you use the environment credentials instead of the organization credentials. For more information about which permissions to use, see xref:api-manager::org-credentials-config-mule4.adoc#obtaining-credentials[obtaining credentials].
 * Cluster-admin role access to the Kubernetes cluster.
 
+
+== Configuring Your Environment Variable 
```

Comment:
I would just put the instructions for linux based OS.

Reply:
@chiragnarangmulesoft do you agree we shouldn't include all OS?

Reply:
Chirag wanted instructions for all OSs. Done! Thanks!